src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -1879,6 +1882,15 @@ private void writeSnapshotManifest(final JSONArray filesJSONArr, final String sn
             {
                 final JSONObject manifestJSON = new JSONObject();
                 manifestJSON.put("files", filesJSONArr);
+                if (ttl != null) {
+                    DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.S'Z'");
```

Comment:
I believe this might be all changed by Instant API just by "Instant.now()"
https://docs.oracle.com/javase/tutorial/datetime/iso/instant.html

        Duration ttl = Duration.ofMinutes(5);
        Instant now = Instant.now();
        String createdAt = now.toString();
        String expireAt = now.plus(ttl).toString();